rust-actix/docker-compose-swag.yml

```diff
@@ -1,19 +0,0 @@
-services:
```

Comment:
Why is this part of the PR?

Reply:
eduardo told me to delete them. They should never have been merged since they fuck with the makefile as they are ready yet